resource-types/typescript-example-website-monitor/src/models.ts

```diff
@@ -137,3 +137,10 @@ export class ResourceModel extends BaseModel {
     }
 }
 
+export class TypeConfigurationModel extends BaseModel {
+    ['constructor']: typeof TypeConfigurationModel;
+
```

Comment:
nit: Looks like extra spaces were added here.

Reply:
This file is auto-generated by the cfn cli.